packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/director_v2/computations.py

```diff
@@ -126,6 +126,7 @@ async def list_computation_collection_runs_page(
     product_name: ProductName,
     user_id: UserID,
     project_ids: list[ProjectID] | None,
+    filter_only_running: bool = False,
```

Comment:
are you using this anywhere else than below? if not I would refrain from using default values.